src/components/Table/hoc/withTableSorting/withTableSorting.scss

```diff
@@ -5,28 +5,27 @@ $block: '.#{variables.$ns}table';
 #{$block} {
     &__sort {
         display: inline-flex;
-        align-items: center;
+        align-items: baseline;
         // `top` to avoid redundant height to appear
         vertical-align: top;
         cursor: pointer;
         user-select: none;
+        border-radius: var(--g-border-radius-xs);
```

Comment:
Whats the use in this?

Reply:
focus ring looks better when it's rounded